Dockerfile

```diff
@@ -1,32 +1,45 @@
 # WebApollo
-# VERSION 2.0
-FROM ubuntu:16.10
-MAINTAINER Eric Rasche <esr@tamu.edu>, Nathan Dunn <nathandunn@lbl.gov>
+# VERSION master
+FROM tomcat:8-jre8
+MAINTAINER Anthony Bretaudeau <anthony.bretaudeau@inra.fr>, Eric Rasche <esr@tamu.edu>, Nathan Dunn <nathandunn@lbl.gov>
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get -qq update --fix-missing && \
-	apt-get --no-install-recommends -y install \
-	git build-essential maven libpq-dev postgresql-common tomcat8 openjdk-8-jdk nodejs npm wget \
-	postgresql postgresql-client xmlstarlet netcat  \
-	zlib1g-dev libexpat1-dev ant curl ssl-cert && \
-	apt-get autoremove -y && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+    apt-get --no-install-recommends -y install \
+    git build-essential maven2 openjdk-8-jdk libpq-dev postgresql-common \
+    postgresql-client xmlstarlet netcat libpng12-dev zlib1g-dev libexpat1-dev \
+    ant perl5 curl ssl-cert && \
+    apt-get autoremove -y && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
the change to spaces seems unnecessary, but not a big deal, just makes the diffs harder to read for this time.